rosbag2_transport/package.xml

```diff
@@ -30,6 +30,7 @@
   <test_depend>rosbag2_compression_zstd</test_depend>
   <test_depend>rosbag2_test_common</test_depend>
   <test_depend>rosbag2_storage_default_plugins</test_depend>
+  <test_depend>rosbag2_storage_mcap</test_depend>
```

Comment:
Still feeling a bit awkward about this, is there a very strong reason we shouldn't add mcap to the `exec_depend` of default plugins? We are planning to ship it, so it doesn't seem to me that there's any harm in doing it.

Or, if we are trying to be very explicit about what we're testing, then maybe `rosbag2_storage_sqlite3` is a `test_depend` here, and `default_plugins` is not.

Reply:
I'm trying to merge these test coverage PRs with the least externally visible disruption possible, which means not adding `rosbag2_storage_mcap` as a default plugin until it actually is one. That might mean a couple of extra lines like this one in some package.xml files which will be reverted later, but I don't think that's too bad.